testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/HttpServerTestOptions.java

```diff
@@ -118,6 +120,12 @@ public HttpServerTestOptions setHasExceptionOnServerSpan(
     return this;
   }
 
+  @CanIgnoreReturnValue
+  public HttpServerTestOptions setHasNetAttributes(boolean hasNetAttributes) {
+    this.hasNetAttributes = hasNetAttributes;
+    return this;
+  }
```

Comment:
Can we leave a `TODO refactor to use httpAttributes` here? I think it's fine to do it like this for now, but long term we'll probably want to use the same mechanism for all http/net attributes